src/components/AddCryptoForm/styles.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components/native';
+
+export const Container = styled.View`
+  display: flex;
+  height: 100%;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const Title = styled.Text`
+  font-size: 25px;
+  font-weight: bold;
+  color: black;
```

Comment:
create a theme file in your root, declare colors and use it for the colors

src/utils/theme.js

const theme = {
colors: {
black: 'black',
white: '#fff'

}
}